### PR TITLE
Remove Py2 deps

### PR DESCRIPTION
**Pull request by Elad-Laufer**
- move more syntax to Py3

### CODE REVIEW

File: media_platform/job/convert_font_job_group.py [hunk 1]
```diff
@@ -1,10 +1,12 @@
...
+
 from media_platform.job.job_group import JobGroup
 
 
 class ConvertFontJobGroup(JobGroup):
 
     @classmethod
     def deserialize(cls, data: dict) -> ConvertFontJobGroup:
-        return super(ConvertFontJobGroup, cls).deserialize(data)
+        return cast(ConvertFontJobGroup, super().deserialize(data))
```
Comment (alon-ne) [inline media_platform/job/convert_font_job_group.py:12]: Looks like all we do is call super, so we can drop this whole method?

File: media_platform/job/replace_extra_metadata_job.py [hunk 2]
```diff
@@ -7,21 +7,18 @@
...
-        # type: (dict) -> ReplaceAudioExtraMetadataSpecification
+    def deserialize(cls, data: dict) -> ReplaceAudioExtraMetadataSpecification:
         return cls(
             Destination.deserialize(data['destination']),
             AudioExtraMetadata.deserialize(data['audioExtraMetadata'])
         )
 
-    def serialize(self):
-        # type: () -> dict
+    def serialize(self) -> dict:
```
Comment (alon-ne) [inline media_platform/job/replace_extra_metadata_job.py:21]:
```suggestion
    def serialize(self) -> Dict:
```

File: media_platform/job/specification.py [hunk 3]
```diff
@@ -1,16 +1,18 @@
-from abc import ABC
+from abc import ABC, abstractmethod
 
 from media_platform.lang.serialization import Serializable, Deserializable
 
 
-class Specification(ABC, Serializable, Deserializable):
+class Specification(Serializable, Deserializable, ABC):
```
Comment (alon-ne) [inline media_platform/job/specification.py:6]: Should ABC be the last base class? If so why?

**Review by alon-ne — CHANGES_REQUESTED** [review]

File: media_platform/job/convert_font_job_group.py (hunk 1 above)
Comment (Elad-Laufer) [reply media_platform/job/convert_font_job_group.py:12]: At the time I wanted it for the hints

File: media_platform/job/specification.py (hunk 3 above)
Comment (Elad-Laufer) [reply media_platform/job/specification.py:6]: Something to do with the way the interpreter sorts the inheritance 'chain', you can only create a new ABC from another if the ABC is always last...

File: media_platform/job/replace_extra_metadata_job.py (hunk 2 above)
Comment (Elad-Laufer) [reply media_platform/job/replace_extra_metadata_job.py:21]: ahhh... man... I'm really not in the mood to do this all over the place...

**Pull request merged**